src/nvim/version_defs.h

```diff
@@ -41,16 +43,5 @@
 // for the ":version" command and "nvim -h"
 #define NVIM_VERSION_LONG "NVIM " NVIM_VERSION_MEDIUM "." STR(NVIM_VERSION_PATCH) NVIM_VERSION_PRERELEASE NVIM_VERSION_BUILD
 
-//
-// Vim version number, name, etc. Patchlevel is defined in version.c.
-//
-#define VIM_VERSION_MAJOR                7
-#define VIM_VERSION_MINOR                4
-#define VIM_VERSION_100     (VIM_VERSION_MAJOR * 100 + VIM_VERSION_MINOR)
-
-// used for the runtime directory name
-#define VIM_VERSION_NODOT       "vim74"
-// swap file compatibility (max. length is 6 chars)
-#define VIM_VERSION_SHORT       "7.4"
 
 #endif  // NVIM_VERSION_DEFS_H
```

Comment:
Is there a need for this file anymore?  I can't imagine why the above defines would not be present.  And even if they could not be--for instance, because we can't figure out the commit or something else--it seems like they could live in version_git.h, lending more credence to calling version_git.h something else.